### PR TITLE
Add some command aliases from the original Elemental Bot

### PR DESCRIPTION
**Pull request by Sightnado**
adds !top and !leaderboard as aliases for !lb and adds !elements as an alias for !inv

### CODE REVIEW

File: eod/cmdhandlers.go [hunk 1]
```diff
@@ -135,12 +135,12 @@ func (b *EoD) cmdHandler(s *discordgo.Session, m *discordgo.MessageCreate) {
 			return
 		}
 
-		if cmd == "inv" {
+		if cmd == "inv" || "elements" {
```
Comment (Nv7-GitHub) [inline eod/cmdhandlers.go:138]: Make sure to do `cmd == "elements"`

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
An issue, seems good otherwise!

**Comment by Sightnado** [comment]
fixed

**Review by Nv7-GitHub — APPROVED** [review]
Seems good!

**Pull request merged**